SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -256,29 +258,24 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 		ComponentDatas.Add(AuthorityIntent::CreateAuthorityIntentData(IntendedVirtualWorkerId));
 	}
 
+#if !UE_BUILD_SHIPPING
```

Comment:
We only spawn the SApatialDebugger in non-shipping, so add the SpatialDebugging mode when in shipping mode is redundant